docs/developer-docs/setup/playground.md

```diff
@@ -0,0 +1,279 @@
+# Motoko playground
+
+## Overview
+
+Motoko playground is a development environment that can be used to build and deploy Motoko canisters directly in the web browser using the Motoko playground canister. Motoko playground can also be run locally, or it can be utilized through the `dfx deploy --playground` command via the CLI. Canisters deployed to the playground use resources that are borrowed from a canister pool. All canisters deployed to the playground will time out after 20 minutes, and new canisters will need to be borrowed for further deployments past the timer. 
+
+Motoko playground canisters provide the same functionality of a canister deployed to the mainnet network. This can be useful for quickly testing canisters and their workflows. 
+
+The Motoko playground is comprised of:
+- The playground's frontend asset canister, which contains:
+    - A Motoko compiler produced by `js-of-ocaml`. 
+    - A Monaco editor that includes support for Motoko syntax validation.
+    - A Vessel package manager that loads libraries from the vessel package set.
+    - A deploy module which integrates the canister's actor class, Candid UI, and upgrade process. 
+- The playground's backend canister that controls all canisters deployed by users on the playground. 
+
+Each canister has an initial amount of 0.5T cycles and can be used for 20 minutes once the canister has been deployed. To avoid wasting cycles, canisters cannot transfer cycles, and the cycles transfer instructions have been removed.
+
+To ensure resources are used fairly by users, proof of work when a user makes a request for a canister ID is required. 
+
+Since Motoko playground is designed for short, small-scale testing, there are limitations and restrictions imposed on canisters deployed to the playground. These limitations are:
+
+- Cycle transfer instructions are silently ignored by the system.
+- Canisters can use at most 1GB of stable memory.
```

Comment:
```suggestion
- Canisters can use at most 1GB of memory.
```
I think it counts heap + stable memory. Canister settings only apply to both combined